results/user-testing-results.md

```diff
@@ -0,0 +1,138 @@
+
+Participants who could not easily see or could not see the staus indicators at all (since they are small and relatively low contrast) relied on prior experience with notebooks, gut instinct about how long that section of code usually takes to run, or the appearance of a cell output to check a cell’s status. Having at least one non-visual option (like an audio indicator) or a visual option that would represent itself in a timely manner for assistive tech (like a notification) could provide users with the support they want. Making these options customizable would be important as well.
+
+#### Flexible command palette search/filter behavior
+
+Because several participants relied heavily on the command palette to complete tasks, the limit of its filtering arose several times. From the user end, the command palette feels like it searches only for direct matches within the command list for whatever the user types in. If a user knows exactly what they are looking for, they often have success. Participants were sometimes able to quickly and flawlessly locate a desired command. But when they were exploring JupyterLab’s command options, unable to remember the exact name of a command, made a typo, or were unfamiliar with a task many participants had to give up using the command palette in order to complete the task. Depending on the assistive tech they use, typos can also be nontrivial to detect for blind or visually impaired users. Having a fuzzy search or other typo/error support rather than only the direct filter would help the command palette continue to be the boon many participants wanted it to be.
+
+## For future tests
+
+Reflecting on the findings and gaps in this set of tests, the following are questions to revisit in the future.
+
+- How might a screen reader user complete common notebook tasks in JupyterLab?
+- What are JupyterLab’s strengths and weaknesses when it comes to authoring notebooks and other documents with assistive tech?
+- What more can we learn about navigating JupyterLab, especially with a variety of assistive tech? What are good examples of navigation that support disabled users?
+- What are the full range of notebook cell output types and what feedback to disabled users have on each type?
```

Comment:
```suggestion
- What are the full range of notebook cell output types and what feedback do disabled users have on each type?
```

Do you mean specifically blind users?

Reply:
I was still thinking of a sample with people who have different disabilities—including but not limited to blindness— when I wrote this. Does that seem to not match the question?

Reply:
Okay, I think the thing that may be throwing you overall in this section is that I would consider it callback-less. Most if not all of these questions I didn't write thinking of something I mentioned above, I wrote them thinking of things we didn't do that I would want to know for all the times people ask me "So is JupyterLab accessible?" A call back approach may be more suited, though. 

I can add a question about better understanding the different ways people find out cell status. I think I'd prefer not to remove this one, though, because the truth is we still haven't tested things like common widgets used in Jupyter notebooks, LaTeX outputs, and some other cell output types.

Reply:
```suggestion
- What are the full range of notebook cell output types and what feedback to disabled users have on each type?
- How do disabled users interact with cell status features? Where does it fail them? What do they do instead?
```

Reply:
You nailed it: it being "callback-less" is what was throwing me, especially because to my brain it _seemed_ that some items in the list were callbacks while others were not. Anyway, don't worry too much about it. It only tripped me up a little bit. It didn't prevent me from making sense of the section or the document as a whole.